package.json

```diff
@@ -1,23 +1,29 @@
 {
-  "name": "satcheljs",
-  "version": "2.1.0",
+  "name": "satcheljs-suite",
+  "version": "2.2.0",
   "description": "Store implementation for functional reactive flux.",
-  "main": "./dist/src/index.js",
   "scripts": {
-    "build": "node_modules/.bin/tsc",
-    "test": "npm run build && jasmine JASMINE_CONFIG_PATH=jasmine.json --verbose"
-  },
-  "author": "Scott Mikula <smikula@microsoft.com>",
-  "license": "MIT",
-  "dependencies": {
-    "mobx": "~2.2.2"
+    "clean": "rimraf dist",
+    "link-all": "node tools/npm-link.js",
+    "publish-all": "node tools/publish-all.js",
+    "normalize": "node tools/normalize-packages.js",
+    "build": "npm run clean && npm run normalize && node tools/build-all.js",
+    "test": "npm run build && npm run link-all && node tools/test-all.js"
   },
   "devDependencies": {
+    "glob": "^7.0.6",
     "jasmine": "~2.4.1",
-    "typescript": "~1.8.9"
+    "mkdirp": "^0.5.1",
+    "rimraf": "^2.5.4",
+    "toposort": "~1.0.0",
+    "typescript": "~2.0.3"
   },
-  "typings": "./dist/src/index.d.ts",
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org/"
```

Comment:
yup - we don't publish THIS repo, we use this repo to npm install globally for the entire set of pkgs